inference-engine/src/vpu/myriad_plugin/myriad_executable_network.cpp

```diff
@@ -113,7 +115,7 @@ void ExecutableNetwork::Import(std::istream& strm,
     }
 
     // TODO: better name
```

Comment:
You changed the related code but didn't resolve TODO.
Please, create a more unique and describing name for an imported network